src/app/consumer.service.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ConsumerService } from './consumer.service';
+import { createService, mockProvider } from '@netbasal/spectator';
+import { ProviderService } from './provider.service';
+import { Subject } from 'rxjs';
+
+describe('ConsumerService', () => {
+  const spectator = createService({
+    service: ConsumerService,
+    providers: [
+      mockProvider(ProviderService, {
+        obs$: new Subject()
```

Comment:
I'm wondering what is the point here as it's already a Subject. That's a dummy example for how to use?

Reply:
Forget about it :)